File: hw/vargas/u3/hw29Singleton/python/ec/edu/espe/singleton/controller/appointment_controller.py
class AppointmentController:
    
    def cancel(self, appointment):
        appointment.status = "Cancelada"
        print(f"Cita {appointment.appointment_id} ha sido cancelada.")

    def mark_completed(self, appointment):
        appointment.status = "Completada"
        print(f"Cita {appointment.appointment_id} marcada como completada.")

File: hw/vargas/u3/hw29Singleton/javaScript/ec/edu/espe/singleton/controller/AppointmentController.js
export class AppointmentController {
    
    cancel(appointment) {
        appointment.status = "Cancelada";
        console.log(`Cita ${appointment.appointmentId} ha sido cancelada.`);
    }

    markCompleted(appointment) {
        appointment.status = "Completada";
        console.log(`Cita ${appointment.appointmentId} marcada como completada.`);
    }
}